ramls/examples/holdingsrecordssource.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
@bohdan-suprun  @jsavell What do you think about using hyphens in the file names for schema and examples?

Reply:
I like this idea.

Reply:
Looking at the other json files in the directory, there are some underscored, some hyphenated, but the vast majority have no separator, which is the 'convention' these new files follow.

Reply:
I don't think there is a convention at the moment. 

I think we should start to introduce one. This pull request seems to be as good a place as any to do so. My preference would be for hyphens.

Reply:
Are you asking me to rename all the files in the directory that aren't already hyphenated as part of this PR?

Reply:
Yes, that is what I'm suggesting. It was a suggestion, if you want to defer or disregard it, that is ok.